### ISSUE TITLE
BarcodeService

### ISSUE THREAD

**Issue by godwinpang** [Honey4Groceries/Honey4Groceries#39]
Find a barcode API that gives us a mapping from barcode to product name.
Then create a BarcodeService just like you used to do yadayadayada.

**Comment by godwinpang** [comment]
https://www.nutritionix.com/business/api

**Comment by paulpan05** [comment]
https://www.programmableweb.com/category/barcodes/api

**Comment by paulpan05** [comment]
https://firebase.google.com/docs/ml-kit/read-barcodes

**Comment by paulpan05** [comment]
We should probably use these two

https://chompthis.com/api/
https://api.cloudmersive.com/docs/barcode.asp

**Comment by godwinpang** [comment]
make a request for the free version of chompthis. They offer it free to .edu emails.

**Comment by paulpan05** [comment]
> make a request for the free version of chompthis. They offer it free to .edu emails.

Do you want to request it instead? It would be easier for you to manage if you use your school email.

**Comment by godwinpang** [comment]
good call. I'll do it then. let's focus on using this API for now - please compile a list of the endpoints that might be useful. If possible, just go ahead and create a free 3 day trial with your credit card and see if you can get some test calls.
